src/rules/noUnusedVariableRule.ts

```diff
@@ -65,7 +65,14 @@ export class Rule extends Lint.Rules.AbstractRule {
     };
     /* tslint:enable:object-literal-sort-keys */
 
-    public static FAILURE_STRING_FACTORY = (name: string) => `Unused variable: '${name}'`;
+    public static FAILURE_TYPE_BIND_ELEM = "binding element";
```

Comment:
I can get behind that, I'm not entirely familiar with the TS syntax this is referring to anyway